src/contracts/LANDRegistry.js

```diff
@@ -22,9 +22,7 @@ export class LANDRegistry extends Contract {
       }
       default:
         throw new Error(
-          'Unknown version when trying to decode land data:',
-          data,
-          '(see https://github.com/decentraland/commons/blob/master/docs/land-data.md)'
+          `Unknown version when trying to decode land data: ${data} (see https://github.com/decentraland/commons/blob/master/docs/land-data.md)`
```

Comment:
can we add quotes? so when `data` is empty it shows like `Unknown version when trying to decode land data: ""`.

Reply:
Yes!